Cargo.toml

```diff
@@ -21,7 +21,7 @@ path = "src/lib.rs"
 all-features = true
 
 [features]
-default = ["bracketed-paste", "events", "windows", "derive-more"]
+default = ["bracketed-paste", "derive-more", "document-features", "events", "windows", "derive-more"]
```

Comment:
It's probably reasonable to move this out of the default features as it's only really useful when compiling the documentation (which enables all-features).